figures/sites.py

```diff
@@ -289,3 +289,31 @@ def site_certificates(site):
             user__organizations__sites__in=[site])
     else:
         return GeneratedCertificate.objects.all()
+
+
+def _get_all_sites():
+    """
+    Return all sites. Do not use this helper directly, but use `get_sites()`.
+
+    Default backend for get_sites() in multi-site mode.
+    """
+    return Site.objects.all()
+
+
+def get_sites():
+    """
+    Get a list of sites for Figures purposes in a configurable manner.
+
+    This functions makes use of the `SITES_BACKEND` setting if configured, otherwise
+    it defaults to  _get_all_sites().
+
+    :return list of Site (QuerySet)
+    """
+    sites_backend_path = settings.ENV_TOKENS['FIGURES'].get('SITES_BACKEND')
```

Comment:
Just to note: Confirming it is ok to assume `settings.ENV_TOKENS['FIGURES']` will always exist.

Figures sets a default empty dict for `settings.ENV_TOKENS['FIGURES']`. See here: https://github.com/appsembler/figures/blob/0.4.dev7/figures/settings/lms_production.py#L80